src/test/app/common/utils/stringUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'chai';
+import { StringUtils } from 'common/utils/stringUtils';
+
+describe('StringUtils', () => {
+  describe('isBlank', () => {
+    it('should return true when value is null', () => {
+      expect(StringUtils.isBlank('')).to.be.true;
+    });
+
+    it('should return true when value is undefined', () => {
+      expect(StringUtils.isBlank(undefined)).to.be.true;
+    });
+
+    it('should return false when value is a space', () => {
+      expect(StringUtils.isBlank(' ')).to.be.false;
+    });
+  });
+});
+
```

Comment:
is there not a library we can use for string utils?

Reply:
not that i am aware of, if you find any, happy to use it.

Reply:
@KiranVarma02 lodash?

Reply:
I did check lodash but missed empty function, I will check

Reply:
Not same thing, is it? I am checking in mobile, I will check later again